src/cpp/include/openvino/genai/tokenizer.hpp

```diff
@@ -165,6 +165,7 @@ class OPENVINO_GENAI_EXPORTS Tokenizer {
     * @param add_special_tokens whether to add special tokens
     * @param max_length optional maximum length to which output will be truncated and/or padded. If not defined, taken from IR.
     * @param pad_to_max_length either pad to max_length, or pad to the longest sequence in the batch. Default is false.
+    * @param padding_side side to pad, either "left" or "right". Default is "right".
```

Comment:
There is no default value. If it's not defined padding side will rely on the value from the IR

Reply:
fixed